packages/launch-screen/package.js

```diff
@@ -6,11 +6,11 @@ Package.describe({
   // between such packages and the build tool.
   name: 'launch-screen',
   summary: 'Default and customizable launch screen on mobile.',
-  version: '1.1.1'
+  version: '1.1.2-beta.0'
```

Comment:
Can we remove these `-beta.0` suffixes from `launch-screen`, `mobile-experience`, and `mobile-status-bar` (commit b8e3d083c9a01306650bb434a7b5c92a390b7aad)?

Also, will the new versions of these packages work with the current version of Cordova, or only with Cordova 9? If they only work with Cordova 9, we should be bumping the minor versions of these packages, so folks using Meteor 1.8.3 won't be exposed to the new versions yet.